### PR TITLE
Update RNN-task.ipynb

### PR DESCRIPTION
**Pull request by DPGoel**
This loss function will perform much better

### CODE REVIEW

**Comment by DPGoel** [comment]
The more accurate formula for cross entropy for binary classification is −(y *log(p)+(1−y)*log(1−p))  not      -y*log(p) .You can just think when the actual answer is 0 ,loss=0 according to you but that is not true

**Comment by MaxPoon** [comment]
I believe we are doing multi-class classification in this assignment. And `−(y *log(p)+(1−y)log(1−p))` is not the correct cross entropy function for multi-class problem. And since we are applying the function `-ylog(p)` to all the classes and sum them up, the loss will always be positive. Close the PR. For more details, see the [discussion](https://datascience.stackexchange.com/questions/9302/the-cross-entropy-error-function-in-neural-networks).
